ext/date/php_date.c

```diff
@@ -497,21 +497,19 @@ timelib_tzinfo *php_date_parse_tzfile_wrapper(const char *formal_tzname, const t
 /* {{{ static PHP_INI_MH(OnUpdate_date_timezone) */
 static PHP_INI_MH(OnUpdate_date_timezone)
 {
-	if (OnUpdateString(entry, new_value, mh_arg1, mh_arg2, mh_arg3, stage) == FAILURE) {
+	DATEG(timezone_valid) = 0;
```

Comment:
I don't think this assignment is needed anymore?

Reply:
And possibly the entire timezone_valid flag?

Reply:
As far as I can tell this is the one place that would assign zero to it, others assign 1 or check for 1. We should probably retarget this change to master though, as there's some minor changes in behavior (as seen in tests).